src/main/target/WARPF7/target.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of INAV.
+ *
+ * INAV is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * INAV is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with INAV.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * This target has been autgenerated by bf2inav.py
+ */
+
+#include <stdint.h>
+
+#include "platform.h"
+
+#include "drivers/bus.h"
+#include "drivers/io.h"
+#include "drivers/pwm_mapping.h"
+#include "drivers/timer.h"
+#include "drivers/pinio.h"
+//#include "drivers/sensor.h"
+
+// Format: DEF_TIM (tim, ch, pin, usage, flags, dmavar)
+timerHardware_t timerHardware[] = {
+    DEF_TIM(TIM1,   CH2N,   PB0,    TIM_USE_OUTPUT_AUTO,    0, 0), // S1
```

Comment:
You may want to switch the timer for S1 and S2 and it conflicts with your led timer.